src/services/book.js

```diff
@@ -0,0 +1,51 @@
+import axios from './axios';
+const booksBase = '/book/register';
+
+export async function createBook(book) {
+  try {
+    const response = await axios.post(booksBase, book);
+    return response.data;
+  } catch (error) {
+    return [];
+  }
+}
+
+const ValidateBook = (book) => {
+  return Object.values(book).every((value) => {
+    if (value === '' || value === null || value === undefined) {
+      return false;
+    }
+    return true;
+  });
+};
+
+export async function collectBookData(method) {
+  try {
+    const book = {
+      title: document.getElementById('BookTitle').value,
+      author: document.getElementById('BookAuthor').value,
+      isbn: document.getElementById('BookISBN').value,
+      category: document.getElementById('BookCategory').innerHTML.toString(),
+      year: document.getElementById('BookYear').getAttribute('value'),
+      picture: document.getElementById('BookPicture').value,
+      editorial: document.getElementById('BookEditorial').value,
+      language: document.getElementById('BookLanguage').innerHTML.toString()
+    };
```

Comment:
Use react state to capture data, all this function should on the form component, only backend request shold be called from outsied the component